pkg/nodeoperations/controller.go

```diff
@@ -114,7 +114,7 @@ func (c *Controller) handleNodeRemoval(ctx context.Context, csibmnodes []nodecrd
 		errors        []string
 		removingNodes []nodecrd.Node
 	)
-	c.log.Info("Starting Node Removal")
+	c.log.Debug("Starting Node Removal")
```

Comment:
This log entry should be moved in actual Node Removal routine.

Reply:
> This log entry should be moved in actual Node Removal routine.

Has changed the code according to your comment.

Reply:
You dont get, this log entry should be moved in the part of code there is actual removal process happen. I think it should be in removeNodes method:

```golang
func (c *Controller) removeNodes(ctx context.Context, csibmnodes []nodecrd.Node) error {
  if len(csibmnodes) > 0 {
    c.log.Debugf("Starting Node Removal, node count: %d",  len(csibmnodes))
  }
// ....
```
As you can see log entry shows only then there are any nodes to remove exist. The same changes should be this `Starting Node Maintenance`. 

After this changes both log entries will not spam log.

Reply:
> You dont get it, this log entry should be moved in the part of code there is actual removal process happen. I think it should be in removeNodes method:
> 
> ```go
> func (c *Controller) removeNodes(ctx context.Context, csibmnodes []nodecrd.Node) error {
>   if len(csibmnodes) > 0 {
>     c.log.Debugf("Starting Node Removal, node count: %d",  len(csibmnodes))
>   }
> // ....
> ```
> 
> As you can see log entry shows only then there are any nodes to remove exist. The same changes should be with `Starting Node Maintenance` log.
> 
> After this changes both log entries will not spam log.

OK. I will change the code according to your comment.